AirCasting/SearchAndFollow/MapSessionsDownloader/APIStruct/MeasurementType.swift

```diff
@@ -0,0 +1,18 @@
+// Created by Lunar on 17/02/2022.
+//
+
+import Foundation
+
+enum MeasurementType: Codable {
+    case particulateMatter
+    case ozone
+}
+
+extension MeasurementType {
+    var name: String {
+        switch self {
+        case .particulateMatter: return "Particulate Matter"
+        case .ozone: return ""
```

Comment:
This file is different than on develop branch, so I think it would be better to sort out this problem with files that shouldn't be included in this PR, because there might be some unwanted changes